themes/survey/vanilla/options/options.js

```diff
@@ -235,7 +235,7 @@ var ThemeOptions = function(){
             //now write the newly created object to the correspondent field as a json string
             $('#TemplateConfiguration_options').val(JSON.stringify(newOptionObject));
             //and submit the form
-            $('#template-options-form').find('button[type=submit]').trigger('click');
+            $('#template-options-form').trigger('submit');
```

Comment:
With trigger('submit') : no HTML5 validation control is done (unsure it's needed here)

Reply:
I am not sure there is a button on that form.
I have run a quick search but couldn't find it.

Reply:
Yes : i think here all is done via JS, then no need to validate form . Just a reminder :)

Reply:
No button , alt solution : add a button in form ?

Reply:
Comment added in the code to explain "no-button" situation. Will be helpfull for future.

Reply:
Tested issue by pulling the PR. Able to save the changes successfully